lib/rust/profiling/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "profiling"
+authors = ["Enso Team <contact@luna-lang.org>"]
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib" ]
+
+[features]
+enable-section-profiling = []
+enable-task-profiling = []
+enable-detail-profiling = []
+enable-debug-profiling = []
+
+default = ["enable-section-profiling","enable-task-profiling","enable-detail-profiling","enable-debug-profiling"]
```

Comment:
let's make it hierarchical. If you set `"enable-detail-profiling"` - all other things (but debug) should be enabled as well. It's like with logging – when you enable debug logs, you still have warning and error logs. But if you set your log levell to warning, then only warning and error logs will be there.

Reply:
Done.

Reply:
If it happens somewhere, why so many flags are left here? Would one not be enough having the hierarchical flag management in place? :)

Reply:
But you still need to be able to address every level? So, there are 4 levels that will activate/deactivate in a hierarchical manner: `enable-section-profiling` will activate everything, `enable-task-profiling` will also activate `detail` and `debug` etc. 
Or am I missing something?

Reply:
Is the `default` not enabling default flags? If so, why do you need to enable so many of them instead of one, which will trigger the rest?

Reply:
I understand now what you mean and removed the extra defaults.